datacenter-deploy-auto-config/secint/consul/server1.json

```
@@ -26,7 +26,7 @@
 		"authorization": {
 			"enabled": true,
 			"static": {
-				"jwt_validation_pub_keys": ["-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEFlUd7FoWSPtRl5maa58XDDjiFplNBxtai41Hq8rnyfQxirYQoLKHKakuZAGpn3PwYewEbYrB+b1f7/P6DzWBRg==\n-----END PUBLIC KEY-----\n"],
+				"jwt_validation_pub_keys": ["-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEFGvHJZCj5SsFlOI48A+Dc4Hezywd\nN8YYjMAQrPTcG6f3jdIEBsh/sVgFTyDYmnwuX6S2ZE3nJ5TtUeJxwwg4EQ==\n-----END PUBLIC KEY-----"],
```

I think you still need the newline \n component in here.
```suggestion
				"jwt_validation_pub_keys": ["-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEFGvHJZCj5SsFlOI48A+Dc4Hezywd\nN8YYjMAQrPTcG6f3jdIEBsh/sVgFTyDYmnwuX6S2ZE3nJ5TtUeJxwwg4EQ==\n-----END PUBLIC KEY-----\n"],
```